rfd/xxxx-azure-cosmos-integration.md

```diff
@@ -0,0 +1,123 @@
+---
+authors: Gabriel Corado (gabriel.oliveira@goteleport.com)
+state: draft
```

Comment:
Could you add reviewers section and please and ask @r0mant and @xinding33 for a review ?